### ISSUE TITLE
Delete the Spaces before pseudo  class 

### ISSUE THREAD

**Issue by hl1221hl** [beautify-web/js-beautify#799]
format<code>div :hover {} </code>to<code>div:hover {} </code>, delete the Spaces before pseudo  class

**Comment by zenyr** [comment]
But...  <http://codepen.io/anon/pen/avXMjN>
````div :hover {color:red} != div:hover {color:red}```` 
:-1:

**Comment by bitwiseman** [comment]
Wait what?   Please provide input, actual output, and expected output.

**Comment by zenyr** [comment]
Sorry @hl1221hl , I kinda misunderstood the original issue. Let me fix my mistake by putting requested data :sob:

## Input
```
div :hover {
  color: red;
}
```
## Output
```
div:hover {
  color: red;
}
```
## Expected
```
div :hover {
  color: red;
}
```

**Comment by hl1221hl** [comment]
@zenyr :+1:

**Comment by bitwiseman** [comment]
Fixed in 1.8.0-rc6
